### ISSUE TITLE
Eviter le spam automatiquement en cas de problème répétitif avec les crons

### ISSUE THREAD

**Issue by targos** [ES-Community/bot#65]
Il y a un tas de solutions différentes possibles. Une des plus simples à implémenter je pense est un petit compteur (in memory) pour chaque cron. Si le run a fail 2 ou 3 fois d'affilée avec la même erreur, on log l'erreur une dernière fois en indiquant dans le message que les erreurs suivantes seront ignorées.
Dès que le run ne fail plus (ou fail avec une erreur différente), le compteur est reset.

**Comment by targos** [comment]
c'est ça!

**Comment by MasterZeno** [comment]
Je m'en occupe

**Comment by Xstoudi** [comment]
Ca serait intéressant de simplement update le premier message de log pour ajouter un truc genre `(x3)`, `(x4)`, etc. si le suivant est identique.

**Comment by MasterZeno** [comment]
Je vais mettre à jour en mettant ta proposition dans la PR

**Comment by Xstoudi** [comment]
Faut pas se presser, il faut laisser aux autres le temps de donner leur avis. 😁

**Comment by MasterZeno** [comment]
Yes, je vais un peu vite :D

**Comment by targos** [comment]
ça m'a l'air d'une bonne idée. Comment on pourrait faire ça? En retenant l'id du message après l'avoir posté ou qch dans le genre?

**Comment by Xstoudi** [comment]
Yes, on le retient pendant le temps T ou on veut pas se faire double-notifier.

**Comment by Xstoudi** [comment]
(ou son hash, peu importe)

**Comment by targos** [comment]
Je demande principalement parce que je ne connais pas trop l'API de Discord. Je ne sais pas ce qu'il faut pour pouvoir mettre à jour un message existant.

**Comment by Xstoudi** [comment]
https://discord.js.org/#/docs/discord.js/main/class/Message?scrollTo=edit

J'pense qu'on peut s'en tirer avec ça.

**Comment by MasterZeno** [comment]
On peut le développer ? :)

**Comment by Xstoudi** [comment]
Je pense que tu peux proposer un truc yes !
